samples/matter/template/sample.yaml

```diff
@@ -3,19 +3,32 @@ sample:
   name: Matter Template
 tests:
   sample.matter.template.debug:
+    sysbuild: true
     build_only: true
     extra_args: CONFIG_NCS_SAMPLE_MATTER_OPERATIONAL_KEYS_MIGRATION_TO_ITS=y
     integration_platforms:
       - nrf52840dk/nrf52840
       - nrf5340dk/nrf5340/cpuapp
       - nrf7002dk/nrf5340/cpuapp
+    platform_allow: nrf52840dk/nrf52840 nrf5340dk/nrf5340/cpuapp nrf7002dk/nrf5340/cpuapp
+    tags: sysbuild
+  sample.matter.template.debug.nrf54l15:
```

Comment:
@nordicjm 
Do you see any issue on your side why the different configuration is needed for nRF54L15 for debug configuration?
Or this is just leftover from old sample.yaml?

Reply:
They are needed for the different build options

Reply:
However on main we have the same configuration for all supported platform
https://github.com/nrfconnect/sdk-nrf/blob/main/samples/matter/template/sample.yaml#L5-L15

Can we align it to the version as it is on main?

Reply:
No, the options for this build differ, they are not done in the main applications prj.conf as it is with child/parent image